benchmark/chart.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import ast
+import csv
+import re
+import sys
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+csv.field_size_limit(sys.maxsize)
+
+root = Path(__file__).parent.absolute()
+
+
+def compare_dataframes(branch1: str, branch2: str) -> None:
+    """Compare generated data frames between branches
+
+    Generates (mostly) the markdown report for the PR comment
+
+    Returns: None
+    """
+    gains = []
+    losses = []
+    dfA = pd.read_csv(Path.joinpath(root, "outputs", f"data-{branch1}.csv"))
+    dfB = pd.read_csv(Path.joinpath(root, "outputs", f"data-{branch2}.csv"))
+
+    # Make the lists equivalent in length
```

Comment:
Why is this needed? Maybe a comment?

Reply:
it shouldn't be ... it was only really needed for testing.